src/app/books/components/book-details/book-details.component.html

```diff
@@ -4,6 +4,12 @@
         <div class="book-info">
             <h2 class="book-info__title">{{ book.title }}</h2>
             <p class="book-info__description">Description: <br>{{ book.description }}</p>
+            Author:
+            <div class="book-info__author">
+                {{ book.author.first_name }}
+                {{ book.author.last_name }}
```

Comment:
@k1k8r will be better to avoid snake case name convention on the FE side and follow the relevant camel case.